sdf.md

```diff
@@ -1056,7 +1050,15 @@ The sdfProperty keyword denotes a group of zero or more Property definitions.
 Properties are used to model elements of state.
 
 The qualities of a Property definition include the data qualities (and
-thus the common qualities), see {{data-qualities}}.
+thus the common qualities), see {{data-qualities}}, additional qualities are shown in {{sdfpropqual}}.
+
+| Quality       | Type      | Description                                             | Default |
+|---------------+-----------+---------------------------------------------------------|---------|
+| (data)        |           | {{data-qualities}}                                      |         |
+| readable      | boolean   | Reads are allowed                                       | true    |
+| writable      | boolean   | Writes are allowed                                      | true    |
+| observable    | boolean   | flag to indicate asynchronous notification is available | true    |
```

Comment:
One thing I noticed here: Should the default `observable` value actually be `true`? In WoT TD, the default value is `false` (see [here](https://w3c.github.io/wot-thing-description/#sec-default-values), and I think this might make more sense in general, as observing should be more complex in many scenarios than a simple read operation.

Reply:
I don't think we want to change the default so late in the game.
You can always set it to false.
Note that the fact that a sensor etc. is observable on the information model level doesn't mean that you actually have a  protocol binding for that.

Reply:
I see, thanks for the feedback :)